spec/models/digital_object_spec.rb

```diff
@@ -361,9 +361,11 @@
           instance.assign_attributes('title' => nil)
           expect(instance).to be_valid
         end
+        it 'is valid when the title only contains a blank non_sort_portion (because blank title fields are removed in before_validation callback)' do
```

Comment:
I found this spec description a little confusing - this may be one of those "too much information" situations, or it may just be a typo. I'll send a commit up with some recommended changes.